cloudomate/hoster/vps/twosync.py

```diff
@@ -195,43 +198,30 @@ def extract_info(cls, url):
         address = str(pages[1]).split('>')[1].split('<')[0]
         return str(amount) + '&' + address
 
-
     @classmethod
-    def _get_hrefs(cls):
-        browser = cls._create_browser()
-        browser.open("https://ua.2sync.org/cart.php")
-        page = browser.get_current_page()
-        hrefs = page.find_all('a', {'class': 'order-button'})
-        lst = [None] * int(len(hrefs)/2)
+    def _convert_bytes_to_gbytes(cls, num):
```

Comment:
Just a sugestion (doesn't have to go to this issue): Wouldn't it be more clear to make one static class with all the conversions? (there is here form bytes but @MattSkala did from MB). I noticed some other classes using the conversions and they are implemented in the same way in several files.
Pls discuss :)

Reply:
I agree. I think we should refactor after the first batch of vps fixes